go/vt/vtctl/workflow/server.go

```diff
@@ -605,7 +605,19 @@ func (s *Server) LookupVindexCreate(ctx context.Context, req *vtctldatapb.Lookup
 
 	lv := newLookupVindex(s)
 
-	ms, sourceVSchema, targetVSchema, cancelFunc, err := lv.prepareCreate(ctx, req.Workflow, req.Keyspace, req.Vindex, req.ContinueAfterCopyWithOwner)
+	var (
+		ms            *vtctldatapb.MaterializeSettings
+		sourceVSchema *topo.KeyspaceVSchemaInfo
+		targetVSchema *topo.KeyspaceVSchemaInfo
+		cancelFunc    func() error
+		err           error
+	)
+	if req.Vindex != nil && len(req.Vindex.Vindexes) > 1 {
+		ms, sourceVSchema, targetVSchema, cancelFunc, err = lv.prepareMultipleCreate(ctx, req.Workflow, req.Keyspace, req.Vindex, req.ContinueAfterCopyWithOwner)
+	} else {
```

Comment:
What if req.Vindex is nil? Also, this is another place where we shouldn't have a separate path IMO. If we support multiple lookup vindexes then we should support that from end to end and we'd have one `prepareCreate` function.

Reply:
Done.